controllers/apply/awards-for-all/lib/lead-time.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+const config = require('config');
+
+module.exports = function getLeadTimeWeeks(
+    country,
+    enableVariableLeadTime = config.get('awardsForAll.enableNewDateRange')
+) {
+    if (enableVariableLeadTime) {
+        if (country === 'england') {
+            return 18;
+        } else {
+            return 12;
+        }
+    } else {
+        return 18;
+    }
```

Comment:
Could you just simplify this to:

`return (enableVariableLeadTime && country !== 'england) ? 12 : 18`

Or are you just aiming to make this more explicit? Fine if so!

Reply:
Yeah just aiming to make it explicit, and the plan is to remove the flagged version pretty soon.